x-pack/auditbeat/module/system/packages/rpm_linux.go

```diff
@@ -0,0 +1,303 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build linux
+
+package packages
+
+import (
+	"fmt"
+	"time"
+	"unsafe"
+
+	"github.com/coreos/pkg/dlopen"
+)
+
+//#include <stdio.h>
```

Comment:
I'm not sure how you feel about it but I wonder if using a block comment `/* */` here would make all this C code more readable.

Reply:
Yeah, I sort of prefer that, just that I didn't see it used in any official examples. I went for it now.